src/Mutation/FileParser.php

```diff
@@ -61,7 +61,7 @@ public function __construct(Parser $parser)
     public function parse(SplFileInfo $fileInfo): array
     {
         try {
-            return $this->parser->parse($fileInfo->getContents());
+            return $this->parser->parse($fileInfo->getContents()) ?? [];
```

Comment:
why that change?

Reply:
This isn't needed, ive instead ignored the error in phpstan, as that can never occur with our settings.